Automattic/jetpack
# Connection: update design in full connection view

#### Changes proposed in this Pull Request:

* Updates the full screen connection view with some design tweaks:
  * New, more consistent looking illustrations.
  * Larger type for better legibility.
  * New subsection headings to highlight each major benefit.

cc @robertbpugh so you can weigh in whether we need better copy as suggested before.

#### Jetpack product discussion

See p6TEKc-3MH-p2 for more details. Kudos @Luchadores for the refreshed design!

#### Testing instructions:

* Spin up this PR on a site that is disconnected. Either start with a brand new site or disconnect your existing site.
* Click on `Jetpack` on the wp-admin sidebar.
* Ensure you see the new design on the full connection view and that everything looks as expected.

#### Screenshots

Before | After
--|--
![image](https://user-images.githubusercontent.com/390760/88914659-d9221300-d25a-11ea-99c1-225d5732db9a.png) |![image](https://user-images.githubusercontent.com/10038976/88925482-e3510b00-d274-11ea-9c37-67b4b0660f3d.png)
![image](https://user-images.githubusercontent.com/390760/89025515-e6a0d100-d31e-11ea-8879-7da1d5f40aef.png) | ![image](https://user-images.githubusercontent.com/390760/89025540-f15b6600-d31e-11ea-839c-3118ab4f0f98.png)



#### Proposed changelog entry for your changes:
* None
